src/common.jl

```diff
@@ -57,3 +57,11 @@ function tiedrank_adj!(ord::AbstractVector, v::AbstractArray)
 end
 
 tiedrank_adj(v::AbstractArray) = tiedrank_adj!(Vector{Float64}(undef, length(v)), v)
+
+# Pool covariance matrices, overwriting the first
+function poolcov!(Sx::AbstractMatrix, nxm1::Int, Sy::AbstractMatrix, nym1::Int)
+    @inbounds for i = eachindex(Sx, Sy)
```

Comment:
Can write this using dot syntax nowadays.

Reply:
Too bad.

The PR looks good to me, but I know so little about this that I don't dare approving.